aviary/models/N3CC/N3CC_data.py

```diff
@@ -1606,7 +1606,7 @@ def _split_aviary_values(aviary_values, slicing):
 landing_fullstop_initial_guesses.set_val('velocity', [135., 0.01], 'kn')
 landing_fullstop_initial_guesses.set_val('mass', detailed_landing_mass, 'lbm')
 landing_fullstop_initial_guesses.set_val('throttle', 0.)
-landing_fullstop_initial_guesses.set_val('angle_of_attack', 0., 'deg')
+landing_fullstop_initial_guesses.set_val(Dynamic.Vehicle.ANGLE_OF_ATTACK, 0., 'deg')
```

Comment:
Looks like you did a global search and replace, which is a good way to do this.  For this N3CC data file though, it looks like we haven't updated any of the other dynamic variables (velocity, mass, etc.).